libvast/src/expression_visitors.cpp

```diff
@@ -578,6 +578,8 @@ bool table_slice_row_evaluator::operator()(const attribute_extractor& e,
   if (e.attr == system::type_atom::value)
     return evaluate(slice_.layout().name(), op_, d);
   // Find the column with attribute 'time'.
+  if (e.attr != system::time_atom::value)
+    return false;
```

Comment:
I was actually suggestion to use `==` instead of `!=`, because this makes adding new paths for new attributes easier:

```cpp
if (e.attr == system::time_atom::value) {
  // ...
}
if (e.attr == system::other_upcoming_attribute::value) {
  // ...
}
return false;
```